src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows;
+using System.Windows.Controls;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
+{
+    /// <summary>
+    /// This is the tag which implements the IntraTextAdornmentTag and is meant to create the UIElements that get shown
+    /// in the editor
+    /// </summary>
+    internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
+    {
+        /// <summary>
+        /// Creates the UIElement on call
+        /// </summary>
+        /// <param name="text">The name of the parameter associated with the argument</param>
+        public InlineParameterNameHintsTag(string text)
+            : base(CreateElement(text), removalCallback: null, (PositionAffinity?)PositionAffinity.Successor)
```

Comment:
1. would be nice to doc why we chose Successor.
2. why do you need the cast to `(PositionAffinity?)`?